RELEASE_LETTERS.md

```diff
@@ -1,3 +1,123 @@
+<h1>Release letter for 6.2.0</h1>
+
+`LogRedactor` is a utility class designed to redact Personally Identifiable Information (PII) from data objects. It provides functions to mask strings, emails, and nested objects while allowing specific properties to be excluded from masking.
+
+- Added maskString method to mask characters between the first and last characters of a string.
+- Added maskEmail method to mask email addresses, preserving the domain.
+- Added isEmail method to identify email addresses.
+- Added shouldExclude method to check if a property should be excluded from masking based on its key path.
+- Enhanced maskPayload method to recursively mask nested objects while respecting the excludeProperties list.
+- Added redact method to serve as the main entry point for redacting PII data in various objects.
+
+These changes ensure sensitive information is properly masked according to specified requirements.
+
+### Import LogRedactor Class
+
+```
+import { LogRedactor } from 'sprinting-retail-common';
+```
+
+### Usage
+
+#### Basic Example
+- The following example demonstrates how to use LogRedactor to redact PII from a data object:
+
+```javascript
+const personCreateBody = {
+  action: 'CreatePerson',
+  id: '87654321-4321-4321-4321-87654321',
+  payload: {
+    id: '12345678-1234-1234-1234-12345678',
+    firstName: 'Testing',
+    lastName: 'Testing',
+    middleName: '',
+    dateOfBirth: '1992-12-20',
+    email: 'testingapi@gmail.com',
+    phone: '123456789',
+    address: {
+      address: 'testinging address',
+      address2: 'testinging address',
+      zipcode: 1234,
+    },
+    countryCode: 'pk',
+    tenantId: 100,
+  },
+};
+
+const redactedData = LogRedactor.redact(personCreateBody);
+
+console.log(redactedData);
+/*
+{
+    "action": "C**********n",
+    "id": "8******************************1",
+    "payload": {
+        "id": "1******************************8",
+        "firstName": "T*****g",
+        "lastName": "T*****g",
+        "middleName": "",
+        "dateOfBirth": "1********0",
+        "email": "t********i@gmail.com",
+        "phone": "1*******9",
+        "address": {
+            "address": "t****************s",
+            "address2": "t****************s",
+            "zipcode": 1234
+        },
+        "countryCode": "p***",
+        "tenantId": 100
+    }
+}
+*/
+```
+
+### Excluding Specific Properties
+- You can also specify properties to exclude from masking by passing an array of property paths to the redact method:
+
+
+```javascript
+
+const excludeProperties = ['action', 'payload.id', 'payload.countryCode', 'payload.consents'];
```

Comment:
I don't know what the scope of this task is, but I find it might be useful to also have a variant where you only mask the properties that are passed in an `includedProperties` parameter.